components/MapOverlay.js

```diff
@@ -12,44 +11,40 @@ import decorateMapComponent, {
   USES_DEFAULT_IMPLEMENTATION,
 } from './decorateMapComponent';
 
-class MapOverlay extends Component {
-
-  static viewConfig = {
-    uiViewClassName: 'AIR<provider>MapOverlay',
-    validAttributes: {
-      image: true,
-    },
-  };
+const viewConfig = {
+  uiViewClassName: 'AIR<provider>MapOverlay',
+  validAttributes: {
+    image: true,
+  },
+};
 
-  static defaultProps = {
-    name: '',
-    rotation: 0,
-    zIndex: 0,
-    transparency: 0,
-    onPress: () => {},
-  };
+const defaultProps = {
+  name: '',
+  rotation: 0,
+  zIndex: 0,
+  transparency: 1.0,
```

Comment:
@Komarev sorry for the confusion, I've mapped it directly to `alpha`. Will fix now.